_docs-v4/custom-views/visibleRange.md

```diff
@@ -28,9 +28,13 @@ You can also specify a function that dynamically generates a range from the curr
 var calendar = new Calendar(calendarEl, {
   defaultView: 'timeGrid',
   visibleRange: function(currentDate) {
+    // Generate a new date for manipulating, based on the current date
```

Comment:
i think it's unnecessary to make copies of the start/end. we can just use the `currentDate` directly in the datemath expressions.